### DISCUSSION TITLE
Sensore Qualità Aria

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2304] [Hardware & Firmware]
Ciao a tutti, 
vorrei inserire due sensori qualità aria che mi rilevassero anche il co2 visto che la mia abitazione è spesso chiusa.
cosa mi potete consigliare? 
ho visto netatmo, il prezzo non è il più basso in assoluto... 
alternative?

**Comment by fizzi** [comment]
Un sensore che utilizzo da tempo è senseair s8 molto preciso ed affidabile.

**Reply by Matty1184** [reply]
> Un sensore che utilizzo da tempo è senseair s8 molto preciso ed affidabile.

non è però un dispositivo fatto e pronto all'uso giusto?

**Reply by fizzi** [reply]
No è il solo sensore, io l'ho collegato ad un D1 mini
